src/Microsoft.AzureHealth.DataServices.Core/Clients/GenericRestClientOption.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Microsoft.AzureHealth.DataServices.Clients
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class GenericRestClientOption : ClientOptions
```

Comment:
I don't think we need this class - are we using any of these properties?

Reply:
I had tried with ClientOptions directly, but I was getting "Constructor on type 'Azure.Core. ClientOptions' not found." error. So I was checking in the core samples, and I found this approach to creating a class that inherits the ClientOptions, and it worked for me. I did some research on the error but couldn't get the proper details, so based on these samples, I applied the logic.